examples/advanced/custom-filters/README.md

```diff
@@ -0,0 +1,61 @@
+## Customize token output using filters
+
+This example shows how you can manage what tokens are generated and how they are organized. This is useful when you want to generate a 1:1 relationship between files and token categories.
```

Comment:
What do you think about changing the name of this example? This example is showing how to generate output files 1:1 based on category/folder. I don't have a good name in mind right now...

Reply:
Darn, I was hoping you had a suggestion haha. I was thinking maybe something like `matching-build-files` ? Open to whatever makes it easier to understand what this example does. :)

Reply:
I'll think about it today..

Reply:
How about "Custom Build", or "Custom Output Structure"?

I'm so excited to see this work, I'm creating a multi-brand design system that has shared global styles among the brands, and specific brand styles for each site. I'd love to output tokens as two separate files, like `global.scss` and `brand.scss`